projects/bp-gallery/src/components/views/exhibitions/exhibition-helper.ts

```diff
@@ -0,0 +1,29 @@
+import { useCreateExhibitionPictureMutation } from '../../../graphql/APIConnector';
+import { FlatPicture } from '../../../types/additionalFlatTypes';
+import { channelFactory } from '../../../helpers/channel-helpers';
+import { MutationFunction } from '../../../types/helper';
+
+export const addExhibitionPicture = async (
+  createExhibitionPicture: MutationFunction<typeof useCreateExhibitionPictureMutation>,
+  exhibitionId: string,
+  selectedPictures: FlatPicture[]
+) => {
```

Comment:
You're always using the same mutation as prop, please just rename this file to exhibition-helper.tsx, use the useCreateExhibitionPictureMutation directly in here and remove the now unnecessary prop, MutationFunction type and the useCreateExhibitionPictureMutations in the two files you used this helper from

Reply:
This now applies to `ExhibitionHelper.tsx` :)

Reply:
I can't convert this to a hook, because I have to call `addExhibitionPicture` inside a function, what's not possible with a hook